### ISSUE TITLE
better support of multiple citekeys within a single :cite:?

### ISSUE THREAD

**Issue by r-xue** [jobovy/sphinx-astrorefs#2]
please see the unexpected behavior in the screenshots.

Not crucial as one can always type single key per `:cite:` multiple times, but it would be nice to have the "multi-citekeys" feature working eventually. I am not sure if a change in `sphinxcontrib-bibtex` is required to achieve this...

---

for the \citep style:

   (:cite:`Barnett:2018aa,Morton:2003aa`)

currently got this:
![multikeys_citep](https://user-images.githubusercontent.com/3730692/91586408-ee9e6180-e91a-11ea-9752-095f77cfbac8.png)

for the \citealt style:

   ::cite:`Barnett:2018aa,Morton:2003aa`

currently got this:
![multikeys_citealt](https://user-images.githubusercontent.com/3730692/91586232-ae3ee380-e91a-11ea-969d-08e8356be9bf.png)

**Comment by jobovy** [comment]
Yeah, I guess this should work, but I'd have to take a look at the HTML it generates to figure out why these aren't getting type-set correctly. As you say, just having separate `:cite:` keys works and this is what I've been doing (because I didn't realize that it was even possible to have multiple keys).

I don't think I'll have time to look into this soon, but if there other people reading this, please upvote 👍 the above comment to indicate that you'd like this support as well.

**Comment by davidorme** [comment]
It would be great to fix this - at the moment, there isn't a way to get (Author 2019, Another & Author 2020). You can get (using MyST not RST, but otherwise the same issue): 

({cite}`Prentice:2013bc`)({cite}`Wang:2017go`) --> (Prentice et al. 2013)(Wang et al. 2017)

or for both:

({cite}`Prentice:2013bc`{cite}`Wang:2017go`)
({cite}`Prentice:2013bc,Wang:2017go`) --> (Prentice et al. 2013Wang et al. (2017))

**Comment by jobovy** [comment]
> It would be great to fix this - at the moment, there isn't a way to get (Author 2019, Another & Author 2020). You can get (using MyST not RST, but otherwise the same issue):
> 
> ({cite}`Prentice:2013bc`)({cite}`Wang:2017go`) --> (Prentice et al. 2013)(Wang et al. 2017)
> 
> or for both:
> 
> ({cite}`Prentice:2013bc`{cite}`Wang:2017go`)
> ({cite}`Prentice:2013bc,Wang:2017go`) --> (Prentice et al. 2013Wang et al. (2017))

So I'm not familiar with MyST, but you can avoid parentheses by adding `:` in front of the cite command (so in RST `::cite:` rather than `:cite:`, perhaps in MyST `:{cite}`?) and then just supply parentheses and any punctuation yourself
```
(:{cite}`Prentice:2013bc`, :{cite}`Wang:2017go`)
```

**Comment by davidorme** [comment]
Just to confirm that does indeed work exactly as you suggest... <looks faintly embarrassed and shuffles off>
